tests/org.gnome.software.gschema.xml

```diff
@@ -0,0 +1,155 @@
+<schemalist>
```

Comment:
Why do we need to copy the GSchema and why do we need it twice? Can't we just assume GS is installed on the system since we need the lib anyway?

Reply:
Doing it twice was just a bug, I guess I did `cp; git add` instead of `git mv`. Initially I did it because I didn't want to depend on GS, which changed afterwards, so I didn't realize. However, it is needed to do some changes to the default schema to make it work well during the tests. I will try to fix that in the tests, so we override the defaults and don't need the schema anymore. Thank you for catching this one!

Reply:
I think the current solution is likely better: 

https://github.com/Cogitri/gnome-software-plugin-apk/pull/31/files#diff-686c499f00ce1b05522a7389cf5379dcf69f681aa35301e28d63aa5fdf758e77R218-R223

Just use the schema from GS and override the values which are not good for tests. Tests have the memory backend configured in meson, so changes shouldn't survive CI if I understand it correctly :)

Reply:
Yup :+1: